controllers/csm_controller_test.go

```diff
@@ -177,6 +177,24 @@ func (suite *CSMControllerTestSuite) TestAuthorizationServerReconcile() {
 	suite.runFakeAuthCSMManager("", true)
 }
 
+func (suite *CSMControllerTestSuite) TestResiliencyReconcile() {
+	suite.makeFakeResiliencyCSM(csmName, suite.namespace, true, append(getResiliencyModule(), getResiliencyModule()...), "powerstore")
```

Comment:
Instead of using a hardcoded name, I will recommend to use the constant defined in types.go for "powerstore".
What do you think about this?